src/bits/bit_vec.rs

```diff
@@ -349,6 +354,23 @@ impl<B: AsRef<[AtomicUsize]>> AtomicBitVec<B> {
         }
     }
 
+    #[inline(always)]
+    unsafe fn swap_unchecked(&self, index: usize, value: bool, order: Ordering) -> bool {
+        let word_index = index / BITS;
+        let bit_index = index % BITS;
+        let data: &[AtomicUsize] = self.data.as_ref();
+
+        let old_value = if value {
```

Comment:
I would call this variable `old_word` because otherwise it looks like `value` and `old_value` have the same type.